Assets/ScriptableObjects/AttackData.cs

```diff
@@ -23,4 +23,6 @@ public class AttackData : ScriptableObject
     public float projectileRange;
     public bool destroyProjectileOnHit;
 
+    public bool possessTarget;
```

Comment:
nit (minor) - personal preference is to name booleans with a prefix `is` or `has` or `did`